content/kubermatic/main/installation/upgrading/upgrade-from-2.27-to-2.28/_index.en.md

```diff
@@ -0,0 +1,86 @@
++++
+title = "[DRAFT] Upgrading to KKP 2.28"
+date = 2025-03-17T00:00:00+02:00
+weight = 10
++++
+
+{{% notice warning %}}
+This is a draft document for an unreleased version of KKP. The information contained here is subject to change and should not be used for production upgrades.
+{{% /notice %}}
+
+{{% notice note %}}
+Upgrading to KKP 2.28 is only supported from version 2.27. Do not attempt to upgrade from versions prior to that and apply the upgrade step by step over minor versions instead (e.g. from [2.26 to 2.27]({{< ref "../upgrade-from-2.26-to-2.27/" >}}) and then to 2.28). It is also strongly advised to be on the latest 2.27.x patch release before upgrading to 2.28.
+{{% /notice %}}
+
+This guide will walk you through upgrading Kubermatic Kubernetes Platform (KKP) to version 2.28. For the full list of changes in this release, please check out the [KKP changelog for v2.28](https://github.com/kubermatic/kubermatic/blob/main/docs/changelogs/CHANGELOG-2.28.md). Please read the full document before proceeding with the upgrade.
+
+## Pre-Upgrade Considerations
+
+{{% notice warning %}}
+Please review [known issues]({{< ref "../../../architecture/known-issues/" >}}) before upgrading to understand if any issues might affect you.
+{{% /notice %}}
+
+### Node Exporter Upgrade (Seed MLA)
+
+KKP 2.28 removes the custom Helm chart for Node Exporter and instead now reuses the official [upstream Helm chart](https://prometheus-community.github.io/helm-charts).
+
+#### Migration Procedure
+
+The following actions are required for migration before performing the upgrade:
+- Replace the top-level key `nodeExporter` with `node-exporter` in the `values.yaml`
+- The key `nodeExporter.rbacProxy` is being removed. For adjusting the kubeRBACProxy container configuration, kindly adjust the values under `node-exporter.kubeRBACProxy` key in `values.yaml` for any customization in place.
+
+Once the above adjustments are done and if you are using it to deploy the Seed MLA components which includes node-exporter, then the KKP `kubermatic-installer` command can be used to perform the migration/upgrade of the existing node-exporter setup to the upstream node-exporter helm chart
+
```

Comment:
```suggestion
Once the above adjustments have been made. If you are using `kubermatic-installer` for the Seed MLA installation, then it will take care of removing the resources for the deprecated node-exporter with the new upstream one by itself.

```